.circleci/config.yml

```diff
@@ -160,11 +158,12 @@ workflows:
           requires:
             - test_unit
 
-      - bintray_publish:
+      - github_publish:
           filters:
             branches:
               only:
                 - release
+                - robby/github-packages
```

Comment:
Is this meant to stay in or just for development?

Reply:
Just for dev, i'll remove it now!